src/formMapper.spec.js

```diff
@@ -193,6 +193,17 @@ describe('map values', () => {
         expect(retVal).toEqual(comparisonVal);
     });
 
+    it('should handle file inputs', () => {
```

Comment:
If this is just a generic test to make sure that formMapper returns an object with an array of file objects for a file input, you could test to see if the return value's `input1` property is an array and that the object (or objects) in the array have some expected file-related properties.